### PR TITLE
Autocomplete: revert breaking change API

### LINKED ISSUE #4156: Breaking changes 1.1 for Autocomplete
Autocomplete of version 1.1 
- does not know the .Focus() any more
- .Close(reason) is now .Close()

### PR DESCRIPTION
**Pull request by stsrki**
Closes #4156

### CODE REVIEW

**Comment by David-Moreira** [comment]
Noticed `SearchChanged` was missing. Certainly to standardize `CurrentSearch` with the new `CurrentSearchChanged`....
But actually, I think `Search + SearchChanged` are more concise naming over `CurrentSearch + CurrentSearchChanged`? 

What do you think? We'd have to obsolete the new `CurrentSearchChanged` instead...

**Comment by David-Moreira** [comment]
Just go ahead and disable the new unit tests or merge it in like this... I'll have to fix them... Manual testing did work at least.

 Visual studio is not collaborating with me... interface super slow to execute tests... jesus.

**Comment by stsrki** [comment]
> Noticed `SearchChanged` was missing. Certainly to standardize `CurrentSearch` with the new `CurrentSearchChanged`.... But actually, I think `Search + SearchChanged` are more concise naming over `CurrentSearch + CurrentSearchChanged`?
> 
> What do you think? We'd have to obsolete the new `CurrentSearchChanged` instead...

I like it more with `Search + SearchChanged` than `CurrentSearch + CurrentSearchChanged`. Should I change it myself? I want to release 1.1.1 today.

File: Source/Extensions/Blazorise.Components/Autocomplete.razor.cs [hunk 1]
```diff
@@ -147,7 +147,11 @@ public override async Task SetParametersAsync( ParameterView parameters )
                 if ( !IsMultiple && CurrentSearch != SelectedText && !string.IsNullOrEmpty( SelectedText ) )
                 {
                     currentSearch = SelectedText;
-                    await CurrentSearchChanged.InvokeAsync( currentSearch );
```
Comment (David-Moreira) [inline Source/Extensions/Blazorise.Components/Autocomplete.razor.cs:150]: Isn't there a SetSearch helper? With both the new parameter and the obsolete?

**Review by David-Moreira — APPROVED** [review]
Looking good. But viewing from mobile... 😅Can you adjust docs?

File: Source/Extensions/Blazorise.Components/Autocomplete.razor.cs (hunk 1 above)
Comment (stsrki) [reply Source/Extensions/Blazorise.Components/Autocomplete.razor.cs:150]: Apparently, there is. You can look at it for 1.1.2?

Comment (David-Moreira) [reply Source/Extensions/Blazorise.Components/Autocomplete.razor.cs:150]: Kk

**Pull request merged**